src/subdataframe/subdataframe.jl

```diff
@@ -131,8 +131,11 @@ index(sdf::SubDataFrame) = index(parent(sdf))
 nrow(sdf::SubDataFrame) = ncol(sdf) > 0 ? length(rows(sdf))::Int : 0
 ncol(sdf::SubDataFrame) = length(index(sdf))
 
+function Base.getindex(sdf::SubDataFrame, colind::Union{ColumnIndex, Colon})
+    return parent(sdf)[sdf.rows, colind]
```

Comment:
Would it make sense to call `view(parent(sdf)[colind], rows(sdf))` here? Of course that wouldn't work for `Colon`.

Also, better use `rows(sdf)` to avoid conflict with `getproperty`: `sdf.rows` no longer works on 0.7.